Project-Hotpot/Project-Hotpot/LoginViewController.swift

```diff
@@ -2,42 +2,71 @@
 //  LoginViewController.swift
 //  Project-Hotpot
 //
-//  Created by Harini Sundaram on 7/8/22.
+//  Created by Harini Sundaram on 7/12/22.
 //
 
 import UIKit
-//import "APIManager.swift"
+import Parse
 
 class LoginViewController: UIViewController {
 
-   // MARK: - Subviews
-   let stackView = UIStackView()
-   let connectLabel = UILabel()
-   let connectButton = UIButton(type: .system)
+    @IBOutlet weak var passwordField: UITextField!
+    @IBOutlet weak var usernameField: UITextField!
+    override func viewDidLoad() {
+        super.viewDidLoad()
 
-   var api_instance = APIManager.shared()
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func didTapRegister(_ sender: Any) {
+        registerUser()
+    }
     
-   // MARK: App Life Cycle
-   override func viewDidLoad() {
-       //initialize APIManager Object
-       
-       super.viewDidLoad()
-       style()
-       layout()
-   }
+    @IBAction func didTapLogin(_ sender: Any) {
+        
+        loginUser()
+    }
     
-    @objc func didTapConnect(_ button: UIButton) {
-        guard let sessionManager = api_instance.sessionManager else { return }
-        sessionManager.initiateSession(with: scopes, options: .clientOnly)
+    func registerUser() {
+        // initialize a user object
+        let newUser = PFUser()
+        
+        // set user properties
+        newUser.username = usernameField.text
+        newUser.password = passwordField.text
         
+        // call sign up function on the object
+        newUser.signUpInBackground { (success: Bool, error: Error?) in
+            if let error = error {
+                NSLog("fuck there was an error registering")
```

Comment:
In general I approve of making things light-hearted and playful, but as your project peer I have to call you out on this particular word; let's keep logs bubbly but professional 😅

Reply:
oh my gosh I'm so sorry, I completely forgot to take this out. It won't happen again.